Makefile

```diff
@@ -17,8 +16,6 @@ REMOTE=origin
 
 ifdef GITHUB_TOKEN
```

Comment:
Should this be an ifdef or should we make it required?

Reply:
The various tasks that need a token use `guard-GITHUB_TOKEN` as a dependency which will cause a fail if the variable is undefined.